webapp/components/Registration/CreateUserAccount.vue

```diff
@@ -109,30 +112,38 @@ export default {
       disabled: true,
       success: null,
       backendErrors: null,
+      checkedDefault: false,
     }
   },
   props: {
     nonce: { type: String, required: true },
     email: { type: String, required: true },
   },
   methods: {
+    checked: function() {
+      this.backendErrors = { message: '' }
+    },
```

Comment:
Nice! I like this. :smile:

Why not set `backendErrors` back to `null`?

And I would find it easier to read if the function was called `resetErrors` – because then I could see right in the template what happens `@change`.

Reply:
I changed it after your suggestion.